ddtrace/debugging/_exception/auto_instrument.py

```diff
@@ -125,10 +133,10 @@ def can_capture(span):
 
     if info_captured is None:
         result = GLOBAL_RATE_LIMITER.limit() is not RateLimitExceeded
-        root.set_tag_str("error.debug-info-captured", str(result).lower())
+        root.set_tag_str(CAPTURE_TAG, str(result).lower())
```

Comment:
Now the root span might not get the `DEBUG_INFO_CAPTURED_TAG` if only child spans have errors. Is this ok?

Reply:
I think so. DEBUG_INFO_CAPTURED_TAG is used by the UI to find spans with error information. If the error didn't propagate to the root span it should not have this tag. 

Am I missing something?